<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMlDl">
    <property name="title">
     <string>ML/DL</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Annotation</string>
     </property>
     <addaction name="actionFileAnnotation"/>
     <addaction name="separator"/>
     <addaction name="actionRoiAnnotation"/>
    </widget>
    <widget class="QMenu" name="menuFeatureExtraction">
     <property name="title">
      <string>Feature Extraction</string>
     </property>
     <addaction name="actionPca"/>
     <addaction name="actionLda"/>
     <addaction name="actionSbs"/>
    </widget>
    <widget class="QMenu" name="menuMl">
     <property name="title">
      <string>ML</string>
     </property>
     <widget class="QMenu" name="menuClassification">
      <property name="title">
       <string>Classification</string>
      </property>
      <addaction name="actionSvm"/>
      <addaction name="actionLogisticRegression"/>
      <addaction name="actionKnn"/>
      <addaction name="separator"/>
      <addaction name="actionDecisionTree"/>
      <addaction name="actionRandomForest"/>
      <addaction name="separator"/>
      <addaction name="actionAdaBoost"/>
      <addaction name="actionGradientBoost"/>
     </widget>
     <addaction name="menuClassification"/>
    </widget>
    <widget class="QMenu" name="menuDl">
     <property name="title">
      <string>DL</string>
     </property>
     <addaction name="actionImageClassification"/>
     <addaction name="actionObjectDetection"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="actionDataset"/>
    <addaction name="separator"/>
    <addaction name="menuDl"/>
    <addaction name="menuMl"/>
    <addaction name="separator"/>
    <addaction name="menuFeatureExtraction"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionDirectX12"/>
    <addaction name="actionOpenGL"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMlDl"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdgeDetector">
   <property name="text">
    <string>Edge Detector</string>
   </property>
   <property name="toolTip">
    <string>Detect Edge</string>
   </property>
  </action>
  <action name="actionFft">
   <property name="text">
    <string>FFT</string>
   </property>
  </action>
  <action name="actionDataset">
   <property name="text">
    <string>Dataset</string>
   </property>
  </action>
  <action name="actionOpticalFlow">
   <property name="text">
    <string>Optical Flow</string>
   </property>
  </action>
  <action name="actionMeanshift">
   <property name="text">
    <string>Meanshift</string>
   </property>
  </action>
  <action name="actionCAMshift">
   <property name="text">
    <string>CAMshift</string>
   </property>
  </action>
  <action name="actionSurfacePlot">
   <property name="text">
    <string>Surface Plot</string>
   </property>
  </action>
  <action name="actionWavelet">
   <property name="text">
    <string>Wavelet</string>
   </property>
  </action>
  <action name="actionLabeling">
   <property name="text">
    <string>Labeling</string>
   </property>
  </action>
  <action name="actionBinarize">
   <property name="text">
    <string>Binarize</string>
   </property>
  </action>
  <action name="actionMorphology">
   <property name="text">
    <string>Morphology</string>
   </property>
  </action>
  <action name="actionLogisticRegression">
   <property name="text">
    <string>Logistic Regression</string>
   </property>
  </action>
  <action name="actionKnn">
   <property name="text">
    <string>kNN</string>
   </property>
  </action>
  <action name="actionGradientBoost">
   <property name="text">
    <string>Gradient Boost</string>
   </property>
  </action>
  <action name="actionDecisionTree">
   <property name="text">
    <string>Decision Tree</string>
   </property>
  </action>
  <action name="actionRandomForest">
   <property name="text">
    <string>Random Forest</string>
   </property>
  </action>
  <action name="actionAdaBoost">
   <property name="text">
    <string>AdaBoost</string>
   </property>
  </action>
  <action name="actionPca">
   <property name="text">
    <string>PCA</string>
   </property>
  </action>
  <action name="actionLda">
   <property name="text">
    <string>LDA</string>
   </property>
  </action>
  <action name="actionSbs">
   <property name="text">
    <string>SBS</string>
   </property>
  </action>
  <action name="actionLut">
   <property name="text">
    <string>Lut</string>
   </property>
   <property name="toolTip">
    <string>Lut</string>
   </property>
  </action>
  <action name="actionNoise">
   <property name="text">
    <string>Noise/Denoise</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionUnsharpMasking">
   <property name="text">
    <string>Unsharp Masking</string>
   </property>
  </action>
  <action name="actionShowHistogram">
   <property name="text">
    <string>Show Histogram</string>
   </property>
   <property name="toolTip">
    <string>Show Histogram</string>
   </property>
  </action>
  <action name="actionSvm">
   <property name="text">
    <string>SVM</string>
   </property>
  </action>
  <action name="actionFileAnnotation">
   <property name="text">
    <string>Simple Annotation</string>
   </property>
  </action>
  <action name="actionRoiAnnotation">
   <property name="text">
    <string>ROI Annotation</string>
   </property>
  </action>
  <action name="actionImageClassification">
   <property name="text">
    <string>Image Classification</string>
   </property>
  </action>
  <action name="actionObjectDetection">
   <property name="text">
    <string>Object Detection</string>
   </property>
  </action>
  <action name="actionShading">
   <property name="text">
    <string>Shading</string>
   </property>
  </action>
  <action name="actionTemplateMatching">
   <property name="text">
    <string>Template Matching</string>
   </property>
  </action>
  <action name="actionHough">
   <property name="text">
    <string>Hough</string>
   </property>
  </action>
  <action name="actionFigureFitting">
   <property name="text">
    <string>Figure</string>
   </property>
  </action>
  <action name="actionKalmanFilter">
   <property name="text">
    <string>Kalman Filter</string>
   </property>
  </action>
  <action name="actionParticleFilter">
   <property name="text">
    <string>Particle Filter</string>
   </property>
  </action>
  <action name="actionDirectX12">
   <property name="text">
    <string>DirectX12</string>
   </property>
  </action>
  <action name="actionOpenGL">
   <property name="text">
    <string>OpenGL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
